tests/API/XCCDF/applicability/test_platform_inheritance.sh

```diff
@@ -1,13 +1,25 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
 set -e
 set -o pipefail
 
 name=$(basename $0 .sh)
-stderr=$(mktemp -t ${name}.out.XXXXXX)
-tmpdir=$(mktemp -d -t ${name}.out.XXXXXX)
-result=$(mktemp -p $tmpdir ${name}.out.XXXXXX)
+
+case $(uname) in
```

Comment:
I agree that it would be good to clean it up. I'll have to think about how to deal with the differences between `mktemp`. For example the FreeBSD version doesn't support `-p` at all and `-t` has different behavior compared to the GNU version.

Reply:
You can change Linux-related code in cases where it makes sense, because there are a lot of differences throughout the test suite where we just want to create a temp file (and it ideally should be identical). Don't over-think it, if test works it is good enough as a unification.

Reply:
I've added those functions to `test_common.sh.in` and switched to using them in the test cases you mentioned. Let me know if it needs to be changed.